package.json

```diff
@@ -29,9 +30,12 @@
     "jshint": ">=2.5.0",
     "karma": "^0.12.31",
     "karma-browserify": "^4.0.0",
+    "karma-chai": "^0.1.0",
     "karma-chrome-launcher": "^0.1.7",
     "karma-firefox-launcher": "^0.1.4",
     "karma-mocha": "^0.1.10",
+    "karma-opera-launcher": "^0.1.0",
```

Comment:
It does!

Not in travis though. In travis we have to stick only with FF and Phantom.

I develop on Mac so I cannot test IE but it would be nice having some test as well and the deps. I may do that later.